### PR TITLE
Bluetooth Random MAC as per issue request #113

### PR DESCRIPTION
**Pull request by danny8972**
A user requested Random MAC addresses to stop MAC based bluetooth limiting.

### CODE REVIEW

**Comment by Ademolght** [comment]
yes please

File: m5stick-nemo.ino [hunk 1]
```diff
@@ -1487,6 +1487,19 @@ void aj_loop(){
...
     deviceType = ajmenu[cursor].command;
     if (maelstrom) {
+      uint8_t newMAC[ESP_BD_ADDR_LEN];
+      
+      // Keep ESP Manufactorer ID 
+      newMAC[0] = 0x02;
+      newMAC[1] = 0xE5;
+
+      // Generate random values for the MAC address and add them to the ESP ID
+      uint32_t randomBytes = esp_random();
```
Comment (Rol3oT) [inline m5stick-nemo.ino:1497]: could we make it a function?

Comment (danny8972) [reply m5stick-nemo.ino:1497]: Definitely

**Comment by danny8972** [comment]
Just updated it as a function would love feedback especially on its placement in the code.

**Comment by Rol3oT** [comment]
looks tidy!

**Comment by Ademolght** [comment]
yes it's good

**Comment by n0xa** [comment]
Awesome, Shipping it!

**Comment by danny8972** [comment]
Thank you :)

**Comment by Ademolght** [comment]
thank you

**Comment by Niximkk** [comment]
not working for me, try some external app to test it, like https://github.com/simondankelmann/Bluetooth-LE-Spam

**Comment by danny8972** [comment]
Random MAC is only triggered for BT maelstrom in my code, and I can confirm through some serial outputs that the MAC address is changing and I have seen an increase in the popups shown on some apple devices. I'll look into the Spam detector that's linked.

**Pull request merged**